sqlglot/dialects/bigquery.py

```diff
@@ -151,6 +151,11 @@ class Parser(parser.Parser):
             TokenType.TABLE,
         }
 
+        def _parse_transaction(self):
+            if self._match_text_seq("TRANSACTION"):
+                return self.expression(exp.Transaction)
+            return self.expression(exp.Command, this=self._prev.text)
```

Comment:
what is this one for?

Reply:
In bigquery BEGIN is valid and different from BEGIN TRANSACTION, as it signifies the start of a block statement.

Reply:
ok put a comment here for that and add a test

Reply:
Done, there's also one test in the parser from before.